scripts/superdesk-search/directives/SearchPanel.js

```diff
@@ -187,6 +187,24 @@ export function SearchPanel($location, desks, privileges, tags, asset, metadata,
                 }
             };
 
+            scope.removeAggregate = function(type, key, evt) {
```

Comment:
Some documentation here would be nice. While it is indeed obvious that an aggregate is removed by calling this function, I think other things happen too (like `$location` changes) and might be worthwhile documenting.